sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/siprouting/implementation/models/SipTrunk.java

```diff
@@ -21,6 +21,31 @@ public final class SipTrunk implements JsonSerializable<SipTrunk> {
      */
     private int sipSignalingPort;
 
+    /*
```

Comment:
We need to be careful to propagate all the properties from public interface, compare with this model:

https://github.com/Azure/azure-sdk-for-java/blob/55700022f8df7302c3c96763092d082434b3518a/sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/siprouting/models/SipTrunk.java

This is the one used in public interface, we need to also surface the properties in that one.